tests/integration/v3_watch_test.go

```diff
@@ -1467,3 +1468,48 @@ func TestV3WatchProgressWaitsForSync(t *testing.T) {
 		t.Fatal("Wrong revision in progress notification!")
 	}
 }
+
+func TestV3WatchProgressWaitsForSyncNoEvents(t *testing.T) {
+	if integration.ThroughProxy {
+		t.Skip("grpc proxy currently does not support requesting progress notifications")
+	}
+	integration.BeforeTest(t)
+
+	clus := integration.NewCluster(t, &integration.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	client := clus.RandClient()
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
The ctx is reused all through this test case, suggest to set a larger timeout value, e.g 5s. arm64 platform isn't stable, it may be flaky on arm64 platform.

Reply:
Right, lowered it for fast iteration when working on the fix, forgot to increase it.